File: src/main/java/com/gioia/radiogio/services/PlayerService.java
package com.gioia.radiogio.services;

import com.gioia.radiogio.data.domains.RadioStation;

interface PlayerService {
    void playRadio(String url);
    void stopRadio();
    void toggleFavourite(Boolean isNowInFavourite, RadioStation radioStation);
    void changeVolume(Integer value);
}

File: src/main/kotlin/com/gioia/radio/services/PlayerService.kt
package com.gioia.radio.services

import com.gioia.radio.data.domains.RadioStation

interface PlayerService {
    fun playRadio(url: String)
    fun stopRadio()
    fun toggleFavourite(isNowInFavourite: Boolean, radioStation: RadioStation?)
    fun changeVolume(value: Int)
}